packages/dx-react-scheduler-material-ui/src/templates/common/select/outlined-select.jsx

```diff
@@ -1,32 +1,40 @@
 import * as React from 'react';
+import { styled } from '@mui/material/styles';
 import * as PropTypes from 'prop-types';
 import Select from '@mui/material/Select';
 import MenuItem from '@mui/material/MenuItem';
 import OutlinedInput from '@mui/material/OutlinedInput';
-import withStyles from '@mui/styles/withStyles';
 
-const styles = ({ typography }) => ({
-  root: {
+const PREFIX = 'OutlinedSelect';
+
+const classes = {
+  root: `${PREFIX}-root`,
+  input: `${PREFIX}-input`,
+  menuItem: `${PREFIX}-menuItem`,
+  inputRoot: `${PREFIX}-inputRoot`,
+};
+
+const StyledSelect = styled(Select)(({ theme: { typography } }) => ({
+  [`&.${classes.root}`]: {
     fontSize: typography.fontSize + 2,
   },
-  input: {
+  [`& .${classes.input}`]: {
```

Comment:
`input` and `inputRoot` classes should be without spacing - They passed to the props

Reply:
Replaced it to StyledOutlinedInput